src/classes/BDI_DataImportService.cls

```diff
@@ -733,7 +764,8 @@ global with sharing class BDI_DataImportService {
     * @param mapDIFieldToDstField The field map from a DI field to the destination field
     * @return boolean True if copied, False is not copied.
     */
-    public static boolean isCopiedDIField(DataImport__c di, String strDIField, SObject objDst, Map<String, String> mapDIFieldToDstField) {
+    public static Boolean isCopiedDIField(DataImport__c di, String strDIField, SObject objDst,
```

Comment:
What is the guideline for the *long* argument list: all of them in a new line, or one line per argument, or split when the line is longer than 120?

Reply:
I tend to split lines more by feel than by guideline. I'm open to suggestions. :)